apps/calendar/js/app.js

```diff
@@ -53,9 +53,16 @@ module.exports = {
     this._routeViewFn = Object.create(null);
     this._pendingManager = new PendingManager();
 
+    var loadedSubViews = false;
+
     this._pendingManager.oncomplete = function onpending() {
       document.body.classList.remove(pendingClass);
       performance.pendingReady();
+      // start loading sub-views as soon as possible
+      if (!loadedSubViews) {
+        loadedSubViews = true;
+        window.require(['lazy_loaded', 'css!lazy_loaded']);
```

Comment:
because the local `require` is synchronous (it's part of the AMD spec).. the async `require` is not part of the AMD spec, so other libs don't implement it.